packages/functional-tests/tests/subscription.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { test, expect } from '../lib/fixtures/standard';
+
+test.describe('severity-1', () => {
+  test('subscribe with credit card and login to product', async ({
+    pages: { relier, login, subscribe },
+  }, { project }) => {
+    test.skip(project.name === 'production', 'prod needs a valid credit card');
+    test.skip(project.name === 'local', 'No need to be run on local');
```

Comment:
nit: Wish there was an easier way to simplify this instead of having to duplicate it for each test, but not sure that the `test.describe` has access to the project names/envs.
Even if we refactored out into a helper function, not sure that helps w/ DRYness or clarity.

Reply:
I know, I tried a few things but nothing worked. So, for now I am keeping it as is but will come back later if I find something and rewrite it.